src/main/java/org/apache/sysml/runtime/controlprogram/paramserv/ParamServer.java

```diff
@@ -67,7 +72,17 @@
 		_model = model;
 		_aggService = new AggregationService(aggFunc, updateType, ec, workerNum);
 		try {
+			Timing tBroad = new Timing(true);
+
 			_aggService.broadcastModel();
```

Comment:
Try to move this time measurement into `broadcastModel` to avoid code duplication.